src/sdk.ts

```diff
@@ -291,3 +295,63 @@ export async function stopTestnet(_testnetToml: Uri) {
         throw new errors.MyError({ Message: "Could not start testnet.", Inner: error });
     }
 }
+
+export async function installRustDebuggerPrettyPrinterScript() {
+    let repository = await showInputBoxWithDefault({
+        title: "Github repository",
+        prompt: "The github repository containing the rust debugger pretty printer script.",
+        defaultInput: "ElrondNetwork/elrond-wasm-rs",
+        ignoreFocusOut: true,
+    });
+    let branch = await showInputBoxWithDefault({
+        title: "Branch",
+        prompt: "The branch to use.",
+        defaultInput: "master",
+        ignoreFocusOut: true,
+    });
+    let inputPath = await showInputBoxWithDefault({
+        title: "File path",
+        prompt: "File path to the pretty printer script.",
+        defaultInput: "tools/rust-debugger/pretty-printers/elrond_wasm_lldb_pretty_printers.py",
+        ignoreFocusOut: true,
+    });
+
+    let url = `https://raw.githubusercontent.com/${repository}/${branch}/${inputPath}`;
+    let prettyPrinterPath = getPrettyPrinterPath();
+    await downloadFile(prettyPrinterPath, url);
+
+    let patch = {
+        "lldb.launch.initCommands": [
+            `command script import ${prettyPrinterPath}`
+        ],
+    };
+    let globalSettingsPath = path.join(os.homedir(), ".config", "Code", "User", "settings.json");
```

Comment:
Hopefully, altering the global settings does not cause unintended effects.

Perhaps we should only alter the workspace settings, as an extra safety measure - at least, for the first release?

Reply:
I think it makes more sense installing this only once, globally, versus installing per workspace (which becomes tedious).
I would keep this as it is.